core/native/refresh/src/main/csharp/Services/IconProvider.Path.cs

```diff
@@ -53,7 +48,7 @@ public T GetPath(Path path, int size)
                 return image;
             }
 
-            var baseImage = GetFileIcon(path.getExtension(), path.isDirectory(), size >= 32, false);
+            var baseImage = GetFileIcon($".{fileIconKey}", path.isDirectory(), size >= 32, false);
```

Comment:
Correct? Before it was the extension only. Now with a leading `.`.

Reply:
Yes, path.getExtension is `ext` instead of `.ext` and SHGetFileInfo doesn't work when leading `.` is missing.